ARKitExample/README.md

```diff
@@ -0,0 +1,19 @@
+# Twilio Video ARKit Example
+
+The project demonstrates how to use Twilio's Programmable Video SDK to stream an augmented reality scene created with ARKit and SceneKit. This example was originally provided as part of a blog post that you can find [here]().
+
+### Setup
+
+See the master [README](https://github.com/twilio/video-quickstart-swift/blob/master/README.md) for instructions on how to generate access tokens and connect to a Room.
+
+This example requires Xcode 9.0, and the iOS 11.0 SDK. An iOS device with an A9 CPU or greater is needed for ARKit to function properly.
+
+### Usage
```

Comment:
I think this detail is important to add to the README.

> For this to work properly **you need to generate a new access token with a different identity** and connect to the same room Arkit using the quickstart app.

Reply:
Fair enough. Let me take one more pass on this section.